cmd/ctr/commands/images/pull.go

```diff
@@ -151,7 +161,9 @@ command. As part of this process, we do the following:
 			return err
 		}
 
-		log.G(ctx).WithField("image", ref).Debug("unpacking")
+		if !context.Bool("no-unpack") {
+			log.G(ctx).WithField("image", ref).Debug("unpacking")
+		}
```

Comment:
We can early-return here if we're not unpacking; the rest of this function deals specifically with unpacking.

```suggestion
		if context.Bool("no-unpack") {
			return nil
		}
```

Reply:
Oh, nice! I assumed `containerd.NewImageWithPlatform` was important.